docs/cli-arguments.md

```diff
@@ -92,6 +92,7 @@
 | `--publish-service-udp` | `namespaced-name` | Service fronting UDP routing resources in "namespace/name" format. The controller will update UDP route status information with this Service's endpoints. If omitted, the same Service will be used for both TCP and UDP routes. |  |
 | `--publish-status-address` | `strings` | Addresses in comma-separated format (or specify this flag multiple times), for use in lieu of "publish-service" when that Service lacks useful address information (for example, in bare-metal environments). | `[]` |
 | `--publish-status-address-udp` | `strings` | Addresses in comma-separated format (or specify this flag multiple times), for use in lieu of "publish-service-udp" when that Service lacks useful address information (for example, in bare-metal environments). | `[]` |
+| `--secret-label-for-caching` | `string` | KIC can only cache secrets resources with specified label. |  |
```

Comment:
The explanation here is not very clear. We should tell the users that:
- If the list is empty, no filters on labels of secrets are applied;
- If the list is non-empty, the secret must have one of the labels with value set to `true`.